### PR TITLE
docs: Added new Advanced Guides section to Pilot's Corner

### LINKED ISSUE #316: Advanced Guides
For advanced A320 features and topics after the beginner guide.

Could take in some of the feature guides (FIX INFO, etc.)

Possible topics see META Documentation Status Quo #73

### LINKED ISSUE #361: Update ECAM A320 Briefing
Would be best to link the old system display pages on the instrument panel to either direct to the ECAM section or provide a link to the new pages. 

image

### PR DESCRIPTION
**Pull request by frankkopp**
closes #316

## Summary
New section for Advanced A320 features not covered in the Beginner Guide. 
Meant as a loose collection of features along the current development. 
More structure can and will be introduced when necessary.

Moved AFloor and Fix Info features guides there.

### Location
Pilot's Corner

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

**Comment by Valastiri** [comment]
need 2 CC the bot team to update links -- unless temporary redirects first

issue created in https://github.com/flybywiresim/discord-bot/

**Comment by Benjozork** [comment]
Make sure we still mention A. FLOOR outside of this / link to the adv guide in common issues

File: docs/pilots-corner/advanced-guides/flight-guidance/overview.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Two types of autopilot and flight director modes are available to guide the aircraft:
+
+### Managed modes:
+
+When the aircraft is using managed targets, the Flight Management and Guidance System (FMGS) guides it along lateral and vertical flight paths and speed profiles computed by the Flight Management function (FM) from data in the MCDU. FM manages the guidance targets.
+
+Managed Modes are activated when the corresponding knob on the FCU is pushed in provided that the aircraft meets the necessary engagement conditions. E.g. Managed Heading (NAV) requires a valid flight plan defined in the MCDU.
+
+### Selected modes
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/flight-guidance/overview.md]: Managed modes has a `:` while selected doesn't -- possible typoe?

File: docs/pilots-corner/advanced-guides/flight-phases.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+The pilot is required to confirm and initiate all descents from cruise altitude by pushing or pulling the ALT knob on the FCU. The managed descent does not occur until the pilot initiates the descent following clearance from ATC.
+
+## APPROACH Phase
+
+The APPROACH phase starts when the pilot activates and confirms the approach on the PERF descent page, or when the approach deceleration pseudo waypoint (DECEL) is passed and the aircraft is below 9500 ft AGL in managed flight.
+
+## GO-AROUND Phase
+
+The GO-AROUND phase is activated when the thrust levers are moved to the TOGA position. The FMS then guides the aircraft through the missed approach procedure.
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/flight-phases.md]: While probably not necessary specify that it's thrust levers to TOGA when on approach?

File: docs/pilots-corner/advanced-guides/protections/afloor.md [hunk 3]
```diff
@@ -10,36 +10,36 @@ This feature is part of our custom autopilot and autothrust system - available i
 
 ### Trigger Conditions
 
-When the angle of attack and airspeed of the aircraft reaches the alpha protection range indicated by the yellow and black band on the PFD speed page. Simply put A.FlOOR will trigger when approaching a stall in the aircraft's current configuration.
+When the angle of attack and airspeed of the aircraft reaches the alpha floor protection range indicated by the yellow and black band on the PFD speed page. Simply put A.FlOOR will trigger when approaching a stall in the aircraft's current configuration.
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/protections/afloor.md]:
```suggestion
When the angle of attack and airspeed of the aircraft reaches the alpha floor protection range indicated by the yellow and black band on the PFD speed page. Simply put, A.FlOOR will trigger when approaching a stall in the aircraft's current configuration.
```

**Review by Valastiri — CHANGES_REQUESTED** [review]

**Comment by Valastiri** [comment]
Forgot to mention -- should we include "feature currently INOP" on some of the protections / features? Mostly like say Go Around is not fully implemented and sometimes requires resetting a CRZ level on the prog page?

**Review by Valastiri — APPROVED** [review]

**Comment by Valastiri** [comment]
- [ ] Fix Info requires a typo change in #overview -> raduises to "radii"

**Pull request merged**